index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./src/styles.css">
+    <title>Bootcamp Adventure</title>
+</head>
+<body>
+    <header>
+        <h1>Survive One Day In A Code Bootcamp!</h1>
+    </header>
+    <main>
+        <h2>Are you ready to start your day? I swear it will be fine!</h2>
+        <form id="sign-up">
+            <p>
+                <label for="name">Enter your name:</label>
+                <input id="name" name="name" required>
+            </p>
+            <fieldset>
+                <h2>Choose your position</h2>
+                <p id="avatar">
+                    <label>
```

Comment:
I would keep convention the same for the use of your label tags. on line 18 you have ```<label for="name">``` but here you wrap your ```<input/>``` element with ```<label></label>```.